README.md

```diff
@@ -201,15 +207,15 @@ After the deployment is complete, you can try the deployed application by naviga
 
 Most of the Azure resources deployed in the prior steps will incur ongoing charges unless removed. Also a few of the resources deployed go into a soft delete status. It's best to purge those once you're done exploring, Key Vault is given as an example here. Azure OpenAI and Azure Machine Learning Workspaces are others that should be purged.
 
+> **_NOTE:_** To purge the AI services resources, `Microsoft.CognitiveServices/locations/resourceGroups/deletedAccounts/delete` permission must be given to the user on the subscription scope where the resources are deployed. A role assignment at the resource or resource group scopes will be insufficient to access the purge functionality. See the [learn documentation](https://learn.microsoft.com/en-us/azure/ai-services/recover-purge-resources?tabs=azure-cli#purge-a-deleted-resource) for additional details.
```

Comment:
How is this not already covered by the permissions listed in the pre-preq?

Reply:
We would just need to be more explicit on the scope.  Since we have them create a resource group that can probably be inferred.  I'll add text to explicitly state that the permissions should be at the subscription level.

Reply:
Yeah, if they cannot create a resource group, they couldn't proceed (it's very intentional that we make that a step, vs saying "choose an existing empty resource group") -- helps us better scope permissions.  If you can still include some bits about the scope in the pre-reqs, that would be great!